api/service.go

```diff
@@ -908,6 +914,15 @@ func (s *service) createBucket(ctx context.Context, proj *c.Project, name string
 		return nil, err
 	}
 
+	if s.dnsManager != nil {
+		parts := strings.SplitN(s.gatewayUrl, "//", 2)
+		if len(parts) > 1 {
+			if _, err := s.dnsManager.NewCNAME(name, parts[1]); err != nil {
+				return nil, err
+			}
+		}
```

Comment:
The `else` case seems like a malformed url, so preferred to ignore?

Reply:
As in, ignore the request?

Reply:
I mean, if `s.gatewayUrl` parsing doesn't have `len(parts)>1`, the current code ignores that and doesn't register the CNAME. Is that something that should happen? I wonder if a malformed `s.gatewayUrl` can backfire somewhere else (or in the future).

Reply:
Oh, I see. Makes sense. We could try to parse the URL on start, that fails -> fatal?

Reply:
It may be an option, yeah. I'll leave if that may worth it to you :P